eZ/Bundle/EzPublishCoreBundle/Command/TrashPurgeCommand.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\API\Repository\Repository;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use eZ\Publish\API\Repository\TrashService;
+
+/**
+ * Console Command which empties eZ Publish trash.
+ */
+class TrashPurgeCommand extends ContainerAwareCommand
```

Comment:
This command is registered in dependency injection container so it should extends `\Symfony\Component\Console\Command\Command` rather then`\Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand`

Reply:
Why don't you use `'@ezpublish.connection'` as the 2nd argument in the service definition and set the `$this->handler` in the constructor?